blocks/carousel/carousel.js

```diff
@@ -199,19 +196,31 @@ const optimizeThumbnails = (picture) => {
     });
 };
 
+// Add P around LI inner text
+const addEnclosingP = (block) => {
+  const listItems = block.querySelectorAll('.carousel > div > div > ul li');
+  listItems.forEach((el) => {
+    const li = el.innerHTML;
+    el.innerHTML = '';
+    el.innerHTML = `<div>${li}</div>`;
```

Comment:
your comment states "add `p` around LI inner text", but you add a `div` around the inner LI?

Reply:
Argh right I will fix it!